------------------------------------------------------------
revno: 14182
revision-id: squid3@treenet.co.nz-20170701232253-3beaysa03xf5c67p
parent: squidadm@squid-cache.org-20170701121615-ktx76udds2mzmc6c
committer: Amos Jeffries <squid3@treenet.co.nz>
branch nick: 3.5
timestamp: Sun 2017-07-02 11:22:53 +1200
message:
  Fix build on FreeBSD after rev.14180
  
  RefCount<> does not support assignment from nullptr with C++03
------------------------------------------------------------
# Bazaar merge directive format 2 (Bazaar 0.90)
# revision_id: squid3@treenet.co.nz-20170701232253-3beaysa03xf5c67p
# target_branch: http://bzr.squid-cache.org/bzr/squid3/3.5
# testament_sha1: d5ecf68c60c022783f69311e9049e546be8fa1a0
# timestamp: 2017-07-01 23:50:58 +0000
# source_branch: http://bzr.squid-cache.org/bzr/squid3/3.5
# base_revision_id: squidadm@squid-cache.org-20170701121615-\
#   ktx76udds2mzmc6c
# 
# Begin patch
=== modified file 'src/http.cc'
--- src/http.cc	2017-07-01 12:08:48 +0000
+++ src/http.cc	2017-07-01 23:22:53 +0000
@@ -1449,7 +1449,7 @@
 
             Comm::ConnectionPointer serverConnectionSaved = serverConnection;
             fwd->unregister(serverConnection);
-            serverConnection = nullptr;
+            serverConnection = NULL;
 
             bool ispinned = false; // TODO: Rename to isOrShouldBePinned
             if (request->flags.pinned) {

